packages/adapters/src/kubernetes-instance-adapter.ts

```diff
@@ -90,14 +95,13 @@ IComponent {
             }
         };
     }
-
-    async run(config: InstanceConfig, instancesServerPort: number, instanceId: string): Promise<ExitCode> {
+    async dispatch(config: InstanceConfig, instancesServerPort: number, instanceId: string, sequenceInfo: SequenceInfo, payload: RunnerConnectInfo): Promise<void> {
         if (config.type !== "kubernetes") {
             throw new Error(`Invalid config type for kubernetes adapter: ${config.type}`);
         }
 
         if (this.adapterConfig.quotaName && await this.kubeClient.isPodsLimitReached(this.adapterConfig.quotaName)) {
-            return RunnerExitCode.PODS_LIMIT_REACHED;
+            throw Error(RunnerExitCode.PODS_LIMIT_REACHED.toString());
```

Comment:
This line probably breaks responses integrity returned from backend.

Check this issue to test response on frontend:
https://github.com/scramjetorg/cloud-platform-panel/issues/524

Old response from server was:
`{"error":"Instance limit reached"}`
Response after changes:
`{"error":"24"}`